src/cms/templates/dashboard/admin_dashboard.html

```diff
@@ -1,5 +1,134 @@
 {% extends "_base.html" %}
 {% load i18n %}
+{% block javascript_head %}
+<script src="{% static 'js/DeleteRowTable.js' %}"></script>
```

Comment:
```suggestion
{% load static %}
{% block javascript_head %}
<script src="{% static 'js/DeleteRowTable.js' %}"></script>
```
You have to load the static tag before you can use it.